packages/js/src/ui/components/Inbox.tsx

```diff
@@ -11,6 +16,8 @@ export type InboxProps = {
   tabs?: Array<{ label: string; value: Array<string> }>;
   mountNotification?: NotificationMounter;
   mountBell?: BellMounter;
+  onNotificationClick?: NotificationClickHandler;
+  onActionClick?: NotificationActionClickHandler;
```

Comment:
❓  Shouldn't we have a `onPrimaryActionClick` and one for the secondary as well?

Reply:
We can have two `onPrimaryActionClick` and `onSecondaryActionClick`, but it's more props, and if we will support X buttons we would have to add X props :/

Reply:
We need to support both buttons now though, right?

Reply:
yeah, I guess it is better to reflect what we have in the API/SDK :)